packages/render-fidelity-tools/test/config.json

```diff
@@ -157,25 +137,206 @@
       }
     },
     {
-      "name": "khronos-Box",
-      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/Box/glTF-Binary/Box.glb",
+      "name": "khronos-BoxAnimated",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/BoxAnimated/glTF-Binary/BoxAnimated.glb",
       "orbit": {
         "theta": 30,
         "phi": 60,
         "radius": 2.38
+      }
+    },
+    {
+      "name": "khronos-CesiumMan",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/CesiumMan/glTF/CesiumMan.gltf",
+      "target": {
+        "y": 0.75
+      },
+      "orbit": {
+        "radius": 2
+      }
+    },
+    {
+      "name": "khronos-Cube",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/Cube/glTF/Cube.gltf",
+      "orbit": {
+        "theta": 30,
+        "phi": 60,
+        "radius": 5
+      }
+    },
+    {
+      "name": "khronos-Duck",
```

Comment:
It appears from looking at the renders that the tone mapping has in fact changed with the upgrade of Filament. Can you show some examples to demonstrate whether it has gotten closer to or farther from the model-viewer and babylon renders?